pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -888,6 +888,12 @@ type DockerSDConfig struct {
 	// The host to use if the container is in host networking mode.
 	// +optional
 	HostNetworkingHost *string `json:"hostNetworkingHost,omitempty"`
+	// Configure whether match the first network if the container has multiple networks defined.
+	// If unset, Prometheus uses its default value.
```

Comment:
Since its a true by default case we should mention about it to avoid surprise

```suggestion
	// If unset, Prometheus uses true by default.
```